src/renderer/components/+extensions/extensions.tsx

```diff
@@ -0,0 +1,110 @@
+import "./extensions.scss";
+import { shell } from "electron";
+import React from "react";
+import { computed, observable } from "mobx";
+import { observer } from "mobx-react";
+import { t, Trans } from "@lingui/macro";
+import { _i18n } from "../../i18n";
+import { Button } from "../button";
+import { WizardLayout } from "../layout/wizard-layout";
+import { Input } from "../input";
+import { Icon } from "../icon";
+import { extensionLoader } from "../../../extensions/extension-loader";
+import { extensionManager } from "../../../extensions/extension-manager";
+
+@observer
+export class Extensions extends React.Component {
```

Comment:
Shouldn't this be a "global" page (like Preferences)?

Reply:
What do you mean? It's kinda "global" page, see `cluster-manager.tsx`:
```
            <Route component={Preferences} {...preferencesRoute} />
            <Route component={Extensions} {...extensionsRoute} />
```

Reply:
It can be, just thinking of a reason to hide clusters and status bar

Reply:
Why don't we hide clusters/status bar here if we do that for Preferences?

Reply:
fixed